src/ExpEvaluator.cpp

```diff
@@ -167,7 +167,7 @@ void ExpEvaluator::initialize() {
   log_printf(INFO, "Initializing exponential interpolation table...");
 
   /* Set size of interpolation table */
-  int num_polar = _polar_quad->getNumPolarAngles();
+  int num_polar = _quad->getNumPolarAngles();
```

Comment:
Here the number of polar angles between `ExpEvaluator` is consistent with `Quadrature`, but this was not the case for `Cmfd` above. I'm sure there is a good reason for this inconsistency, but it should be detailed in some comments.